crm_quotation_link/models/crm_lead.py

```diff
@@ -0,0 +1,75 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, _
+
+
+sales_order_states = [
+    'progress', 'manual', 'shipping_exept', 'invoice_except', 'done']
+
+quotations_states = ['draft', 'sent', 'waiting_date']
+
+
+class crm_lead(models.Model):
+    _inherit = 'crm.lead'
+
+    def count_sales_order(self):
```

Comment:
But if you want correct data, you have to do that, because cache mechanism will not be alerted when new orders are added.